src/query/query_manager.c

```diff
@@ -1776,6 +1776,8 @@ xqmgr_prepare_and_execute_query (THREAD_ENTRY * thread_p, char *xasl_stream, int
   bool saved_is_stats_on;
   bool xasl_trace;
 
+  static int qmgr_max_query_entry_per_tran = prm_get_bool_value (PRM_ID_QMGR_MAX_QUERY_PER_TRAN);
```

Comment:
I think prm_get_integer_value() should be used.

Reply:
fixed. thanks.